### ISSUE TITLE
[20220501] Weekly AI ArXiv 만담 

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#50]
### News
- Conference
  -  ICLR 2023: 르완다! (케냐, 탄자니아 서쪽, 콩고민주공화국 동쪽)
![image](https://user-images.githubusercontent.com/11782739/166086698-b848f6aa-c5fb-48fe-b337-76ee61b5d110.png)
  - [HyperscaleFAccT CRAFT at ACM FAccT 2022](https://naver-career.gitbook.io/en/teams/clova-cic/events/hyperscalefacct-facct-2022)
  - [NeurIPS Affinity Group Workshop](https://neurips.cc/Conferences/2022/CallForAffinityWorkshopsAndSocials): 5월 27일 데드라인
- [Large LM 이 과학연구에 끼칠 수 있는 영향](https://www.nature.com/articles/d41586-022-01191-3)
- [NVidia H100 가격이.... $30,000 이상](https://www.tomshardware.com/news/nvidia-hopper-h100-80gb-price-revealed)

### Arxiv
- [Flamingo: a Visual Language Model for Few-Shot Learning](https://www.deepmind.com/blog/tackling-multiple-tasks-with-a-single-visual-language-model)
  - Deepmind에서 공개한 prompt 기반의 in-context zero/few-shot learning이 가능한 visaul-text foundation model.
  - Visual encoder (Frozen NFNet: Normalization-free ResNet) +  LLM (Frozen ChinChilla) + Perceiver Resampler + Gated cross-attention-dense layer 로 구성
  - Perceiver Resampler의 역할은 유동적인 input 크기 (이미지, 비디오)를 작은 크기의 latent dimension 으로 줄여서 LM의 입력으로 쓸 때 computational overhead 감소
  - Gated cross attention-ffw 는 Vision encoder-Perceiver Resampler를 통해 들어온 visual feature 를 Frozen LM에 잘 밀어넣기 위한 모듈로 초기엔 tanh gate에 의해 거의 0으로 시작 그리고 점점비중이 높아짐.
  - Pretraining data: ALIGN, M3W, LTIP, VTP 활용, In-context few-shot downstream 매우 다양
  - 크기는 3B, 9B, 80B, 3개 모델 모두 NFNet은 435M NFNet 사용, 80B이 경우 70B Chinchilla와 추가 파라미터 10B
  - 색깔 맞추기 대화
https://user-images.githubusercontent.com/11782739/166086527-424e3d73-2447-4fce-a3b0-62d295f00271.mp4
![image](https://user-images.githubusercontent.com/11782739/166138394-1cf03e83-6a4f-4572-943e-71077003f472.png)
![image](https://user-images.githubusercontent.com/11782739/166138402-e5b089af-50e9-40ac-bd13-aa29db719537.png)
![image](https://user-images.githubusercontent.com/11782739/166138437-e7b4599b-7608-4fe5-ac52-e1177bb5e5c9.png)
![image](https://user-images.githubusercontent.com/11782739/166138509-5cd3663e-ba92-47aa-9e30-c23faedeb482.png)

- 흥미있는 연구들
  - [Why does Self-Supervised Learning for Speech Recognition Benefit Speaker Recognition?](https://arxiv.org/abs/2204.12765)
    - 제목 그대로 음성인식용 SSL이 화자인식에 도움을 주는데에 대한 심도있는 분석: mask prediction loss, data 양, 모델 크기가 중요하더라.
  - [Standardized Evaluation of Machine Learning Methods for Evolving Data Streams](https://arxiv.org/abs/2204.13625)
    - 계속 분포가 변하는 데이터 학습 및 모델 성능 평가 프로토콜을 교통정리한 연구 (from U of Tuebingen)
    - https://github.com/haugjo/float
  - [Generative Adversarial Networks for Image Super-Resolution: A Survey](https://arxiv.org/abs/2204.13620)
    - GAN 기반의 SR연구에 관심있는 분들 입문서로 활용하시면 좋겠어요

**Comment by ghlee0304** [comment]
### Arxiv

- 흥미있는 연구들 
    - [Pseudo strong labels for large scale weakly supervised audio tagging](https://arxiv.org/abs/2204.13430)
    - [Unaligned Supervision For Automatic Music Transcription in The Wild](https://arxiv.org/pdf/2204.13668.pdf)
        - 샘플 URL : https://benadar293.github.io/
    - [Music Enhancement via Image Translation and Vocoding](https://arxiv.org/abs/2204.13289)

**Comment by veritas9872** [comment]
**OpenGlue: Open Source Graph Neural Net Based Pipeline for Image Matching**

![image](https://user-images.githubusercontent.com/33523965/166144078-5cf88a09-d4a4-4e2d-9a01-175af862381a.png)

Kaggle: https://www.kaggle.com/datasets/ksork6s4/openglue
Arxiv: https://arxiv.org/abs/2204.08870
GitHub: https://github.com/ucuapps/OpenGlue


**Understanding The Robustness in Vision Transformers**

![image](https://user-images.githubusercontent.com/33523965/166144214-8f5601e6-8006-481c-bf9b-d5107ce3d442.png)

Arxiv: https://arxiv.org/abs/2204.12451
GitHub: https://github.com/NVlabs/FAN

**Comment by hollobit** [comment]
### ISO/IEC JTC 1/SC 42(Artificial Intelligence) plenary week (4/18 ~ 4/30) 
https://www.iso.org/committee/6794475.html
- 참여자 240명, 11개 표준 제정, 26개 표준 개발 중 
   * 3개 timezone rotation : 아침6시, 오후14시, 오후22시 
   * WG1: Foundational. WG2: Data, WG3 : Trustworthiness, WG4: Application, WG5: Computational Approach 
- 선도그룹: 미국, 중국, 일본, 영국
   * JWG 2 활동 시작: Testing of AI-based system (영국, 한국) 
- 추격그룹: 인도, 프랑스, UAE
   * UAE: "Standards for Unsupervised Learning" - unsupervised + RL + decision transformer" 
   * UAE: "Standards for Explainability in AI" 
   * 인도 : verification and validation analysis of AI systems" 
   * 프랑스 : AG3 
- 배후세력 

### Google FormNet
https://ai.googleblog.com/2022/04/formnet-beyond-sequential-modeling-for.html
- [ACL 2022](https://www.2022.aclweb.org/) 에서 발표된 “ [FormNet: Structural Encoding Beyond Sequential Modeling in Form Document Information Extraction](https://arxiv.org/pdf/2203.08411.pdf) ”
- 문서 정보 추출을 위한 양식의 차선의 직렬화를 완화하기 위해 FormNet이라는 구조 인식 시퀀스 모델을 제안
- Rich Attention ( RichAtt ) 메커니즘, ETC(Extended Transformer Construction) + BERT, 다국어 어휘, OCR, GCN
- CORD 벤치 마크에서 F1 score 
![](https://blogger.googleusercontent.com/img/b/R29vZ2xl/AVvXsEjc8QgZOGd-NciNi4jhlYVA4SsrVL04Bra_pUx2Lr1en34IFSqU1mJ_ol0TTomqvlUV4Id924PKe2a43QuWni7u9ZgfNLMsVVJxqWmwFRsZERxLyjbF29njjXVsDxPYIFDA7wFzBOJeQiymiuANsZV4zWt7mH-hm-UGdZVWgv-sx8nesZpEFgztXHT8AA/s1074/image3.png)

**Comment by jwlee-ml** [comment]
### DeiT III: Revenge of ViT (밀린 숙제)
- [https://arxiv.org/abs/2204.07118](https://arxiv.org/abs/2204.07118)
- ViT 계의 [ResNet strikes back](https://arxiv.org/abs/2110.00476) 같은 논문
- 흥행에는 실패한 것 같기도?
- 몇가지 Training Technique을 활용해서 vanilla ViT 성능을 끌어올린(쥐어짠) 논문
- 사용한 방법들
Stochastic Depth, LayerScale, Binary Cross Entropy, Data Augmentation, Croping
